insights/specs/default.py

```diff
@@ -330,8 +330,7 @@ def semid(broker):
                                  simple_command("/usr/bin/lsinitrd -f /etc/lvm/lvm.conf")
                                  ])
     lsmod = simple_command("/sbin/lsmod")
-    lspci = simple_command("/sbin/lspci")
-    lspci_kernel = simple_command("/sbin/lspci -k")
+    lspci = first_of([simple_command("/sbin/lspci -k"), simple_command("/sbin/lspci")]),
```

Comment:
@vishwanathjadhav this should be a `simple_command` instead of `first_of` to be consistent with the changes you are making in https://github.com/RedHatInsights/insights-frontend-assets/pull/110.

Reply:
Done